test/import/import-k8s-util.test.ts

```diff
@@ -142,7 +142,7 @@ describe('safeParseJsonSchema', () => {
             },
           },
           required: [
-            'sideEffects', 'not a word',
+            'sideEffects', 'invalid\tvalue',
```

Comment:
I don't like that spaces are now allowed in the `required` key. This array has to be a list of property names, and property names cannot contain spaces...

Why not apply the exception specifically on the `enum` key? Since this is security related, I would much prefer to scope it down as much as possible, and handle the next scenario as it comes along, not beforehand.